Update readme (#85)
memorysafety/zlib-rs: README.md

> # zlib-rs: a safer zlib
> This repository contains a zlib API compatible Rust implementation of the zlib
> file format. There are two public crates available:

Reviewer: ```suggestion
# zlib-rs: a safer zlib

This repository contains a Rust implementation of the zlib file format that is compatible with the zlib API. 

This repository contains two public crates:
```

* `zlib-rs`, a Rust implementation based on [zlib](https://www.zlib.net/manual.html)
  with a safe rust API

> * `libz-rs-sys`, an unsafe zlib compatible C API for usage in non-Rust
>   applications

Reviewer: ```suggestion
* `libz-rs-sys`, an unsafe zlib-compatible C API for usage in non-Rust
  applications
```
